### PR TITLE
SRM release notes for release 803

### PR DESCRIPTION
**Pull request by SudheendraKatte**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: release-notes/service-reliability-management.md [hunk 1]
```diff
@@ -18,42 +18,48 @@ Review the notes below for details about recent changes to Harness Service Relia
...
 
 
-```mdx-code-block
-  </TabItem>
-  <TabItem value="Fixed issues">
-```
+### Fixed issues
 
-This release does not include any fixed issues.
+- Users were unable to update the change source configuration within their monitored service.(SRM-15083)
```
Comment (deepakbolangady) [inline release-notes/service-reliability-management.md:34]: Unless the audience for this note is not the user who experienced this issue, can we write in the second person? Retain if the reader is, say, an account admin who reported the issue on behalf of their users.

File: release-notes/service-reliability-management.md [hunk 2]
```diff
@@ -18,42 +18,48 @@ Review the notes below for details about recent changes to Harness Service Relia
...
+  
+  This issue has been resolved. Users can now update the change source configurations in their monitored service.
```
Comment (deepakbolangady) [inline release-notes/service-reliability-management.md:36]: Plural "configurations"? The issue description has the singular. Even if it's possible to have more than one change source config, I just thought I'll let you know that the difference stood out to me.

**Review by deepakbolangady — APPROVED** [review]
Approved with a couple of comments. Thanks.

File: release-notes/service-reliability-management.md (hunk 2 above)
Comment (SudheendraKatte) [inline release-notes/service-reliability-management.md:36]:
```suggestion
  This issue has been resolved. You can now easily update the change source configurations within your monitored service.
```

File: release-notes/service-reliability-management.md (hunk 1 above)
Comment (SudheendraKatte) [inline release-notes/service-reliability-management.md:34]:
```suggestion
- Unable to update the change source configuration within their monitored service. (SRM-15083)
```

**Review by deepakbolangady — APPROVED** [review]

**Pull request merged**